src/test/regress/expected/isolation_get_distributed_wait_queries_mx.out

```diff
@@ -212,7 +212,7 @@ step s3-select-distributed-waiting-queries:
 
 blocked_statement         |current_statement_in_blocking_process|waiting_node_name|blocking_node_name|waiting_node_port|blocking_node_port
 ---------------------------------------------------------------------
-UPDATE tt1 SET value_1 = 5|UPDATE tt1 SET value_1 = 4           |localhost        |localhost         |            57638|             57637
+UPDATE tt1 SET value_1 = 5|UPDATE tt1 SET value_1 = 4           |coordinator_host |coordinator_host  |            57636|             57636
```

Comment:
these changes are not our favorite ones with @halilozanakgul.

The point is that with gpid infrastructure, the queries running with `run_commands_on_session_level_connection_to_node` shows up part of the coordinator gpid, hence we saw coordinator host.

We tried few alternatives, but all seemed too costly, we are OK to skip this for now.

In the future, we might considerer improving this